From: Technion <Technion>
Date: Mon, 11 Feb 2013 22:08:53 +1100
Subject: [PATCH] Changed wording on logging. Previously, we received errors
 like this: mmc0: could read SD Status register (SSR) at the 3th attempt A
 more sensible response is now returned. A typo also fixed in comments.

---
 drivers/mmc/core/sd.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/core/sd.c b/drivers/mmc/core/sd.c
index 1ee6cf3..90228f9 100644
--- a/drivers/mmc/core/sd.c
+++ b/drivers/mmc/core/sd.c
@@ -238,7 +238,7 @@ static int mmc_read_scr(struct mmc_card *card)
 
 		err = mmc_app_send_scr(card, card->raw_scr);
 		if( !err )
-			break; // sucess!!!
+			break; // success!!!
 
 		touch_nmi_watchdog();	  // we are still alive!
 
@@ -320,7 +320,7 @@ static int mmc_read_ssr(struct mmc_card *card)
 
 	if( tries > 1 )
 	{
-		pr_info("%s: could read SD Status register (SSR) at the %dth attempt\n", mmc_hostname(card->host), tries );
+		pr_info("%s: read SD Status register (SSR) after %d attempts\n", mmc_hostname(card->host), tries );
 	}
 
 	for (i = 0; i < 16; i++)
